src/engine/bank03.asm

```diff
@@ -2873,14 +2871,13 @@ ScriptCommand_ShowSamTutorialMultichoice: ; d317 (3:5317)
 	set_flag_value EVENT_FLAG_75
 	jp IncreaseScriptPointerBy1
 
-.multichoice_menu_args ;d32b
+.multichoice_menu_args: ; d32b (3:532b)
```

Comment:
I don't think the : is supposed to be there. This is an artifact of the way I merged